coalib/coala_modes.py

```diff
@@ -64,7 +64,7 @@ def mode_json(args, debug=False):
 
     if args.output:
         filename = str(args.output[0])
-        with open(filename, 'w+') as fp:
+        with open(filename, 'w') as fp:
```

Comment:
actually, do we wanna test that? Simply write to the same file twice, and check if content was replaced and not appended^^

Reply:
IMO, it doesn't make sense to add tests for this as its python internals. Just for safety, I've tested it locally and seems to be working fine.

Reply:
>  it doesn't make sense to add tests for this as its python internals.

It definitely reflects on the user. That's why we should test it.
However, let's open an issue for that afterwards, imo not so important right now as it's currently not officially standardized I believe whether we append or overwrite output files.